doc/governance.rst

```diff
@@ -20,95 +20,107 @@ the project community.
 Roles And Responsibilities
 ==========================
 
+We distinguish between contributors, core contributors, and the technical
+committee. A key distinction between them is their voting rights: contributors
+have no voting rights, whereas the other two groups all have voting rights,
+as well as permissions to the tools relevant to their roles.
+
 Contributors
 ------------
 
 Contributors are community members who contribute in concrete ways to the
 project. Anyone can become a contributor, and contributions can take many forms
 – not only code – as detailed in the :ref:`contributors guide <contributing>`.
+There is no process to become a contributor: once somebody contributes to the
+project in any way, they are a contributor.
```

Comment:
Should we give examples of (non-technical) contributions?

Reply:
One goal in the governance has been to keep it short, otherwise it'd be very long if we want to include everything here. The contribution guideline has a list of possible contributions, the same way that SLEP000 explains how we do sleps.

Reply:
I'm in favour of keeping it short. Maybe turning "contributes to the project" into a link to the contrib docs is a compromise (though I worry about the link becoming out of date, but maybe it is a "premature optimisation" kinda worry  :-/)

Reply:
There is already the link to the contributing guide here, right before the lines I added.